target/linux/ipq40xx/patches-5.4/902-dts-ipq4019-ap-dk04.1.patch

```diff
@@ -116,17 +117,17 @@
  				spi-max-frequency = <24000000>;
  			};
  		};
+@@ -103,9 +129,48 @@
+ 			perst-gpio = <&tlmm 38 0x1>;
+ 		};
  
--		pci@40000000 {
--			status = "ok";
--			perst-gpio = <&tlmm 38 0x1>;
```

Comment:
This looks like a _change_ to me, i.e. not removing pci@40000000 anymore?

Reply:
Yes, it made no sense in dropping the PCI node as DK04.1 has PCI-E slot as default.

Reply:
I essentially just removed it when migrating from files to patch in order to have the exact same nodes (so, I didn't need to test it). While I generally agree that it's probably more correct to have the pci node, I wouldn't add it as a by-product here, but would prefer to have it in a separate commit.

Reply:
Ok, will move it as a separate thing